openshift/topics/get_started.adoc

```diff
@@ -20,14 +20,23 @@ To create a Registry Service Account, navigate to the link:https://access.redhat
 See the link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication] article for more information.
 ====
 
-To update to the latest {project_openshift_product_name} images, run the following commands:
+. Ensure that you are logged in as a cluster administrator or a user with project administrator access to the global openshift project:
 
-. On your master host(s), ensure that you are logged in as a cluster administrator or a user with project administrator access to the global `openshift` project.
+. Choose a command based on your version of OpenShift Container Platform.
+
+.. If you are running an OpenShift Container Platform v3 based cluster instance on (some) of your master host(s), perform the following:
 +
 [source,bash,subs="attributes+,macros+"]
 ----
 $ oc login -u system:admin
 ----
+.. If you are running an OpenShift Container Platform v4 based cluster instance, link:https://docs.openshift.com/container-platform/latest/cli_reference/openshift_cli/getting-started-cli.html#cli-logging-in_cli-developer-commands[log in to the CLI] as the link:https://docs.openshift.com/container-platform/latest/authentication/remove-kubeadmin.html#understanding-kubeadmin_removing-kubeadmin[kubeadmin] user:
++
+[source,bash,subs="attributes+,macros+"]
+----
+$ oc login -u kubeadmin -p removing-kubeadmin[kubeadmin_password] \https://openshift.example.com:6443
```

Comment:
Drop the `removing-kubeadmin[kubeadmin_password]` and replace it with `-p password`.

Reply:
Just corrected this also.